packages/localizations/src/el-GR.ts

```diff
@@ -0,0 +1,692 @@
+          subtitle__regenerate:
+            'Λάβετε ένα νέο σετ ασφαλών εφεδρικών κωδικών. Οι προηγούμενοι εφεδρικοί κωδικοί θα διαγραφούν και δεν μπορούν να χρησιμοποιηθούν.',
+          actionLabel__regenerate: 'Επαναδημιουργία κωδικών',
+        },
+        totp: {
+          headerTitle: 'Εφαρμογή αυθεντικοποίησης',
+          title: 'Προεπιλεγμένος παράγοντας',
+          subtitle:
+            'Αυτός ο παράγοντας θα χρησιμοποιηθεί ως ο προεπιλεγμένος τρόπος αυθεντικοποίησης δύο βημάτων κατά την είσοδο.',
+          destructiveActionTitle: 'Αφαίρεση',
+          destructiveActionSubtitle:
+            'Αφαιρέστε την εφαρμογή αυθεντικοποίησης από τις μεθόδους αυθεντικοποίησης δύο βημάτων',
+          destructiveActionLabel: 'Αφαίρεση εφαρμογής αυθεντικοποίησης',
+        },
+      },
+      activeDevicesSection: {
+        title: 'Ενεργές συσκευές',
+        primaryButton: 'Ενεργές συσκευές',
+        detailsTitle: 'Τρέχουσα συσκευή',
+        detailsSubtitle: 'Αυτή είναι η συσκευή που χρησιμοποιείτε αυτήν τη στιγμή',
+        destructiveActionTitle: 'Αποσύνδεση',
+        destructiveActionSubtitle: 'Αποσυνδεθείτε από τον λογαριασμό σας σε αυτήν τη συσκευή',
+        destructiveAction: 'Αποσύνδεση από τη συσκευή',
+      },
+      web3WalletsSection: {
+        title: 'Πορτοφόλια Web3',
+        primaryButton: 'Πορτοφόλια Web3',
+        destructiveActionTitle: 'Αφαίρεση',
+        destructiveActionSubtitle: 'Αφαιρέστε αυτό το πορτοφόλι Web3 από τον λογαριασμό σας',
+        destructiveAction: 'Αφαίρεση πορτοφολιού',
+      },
+      dangerSection: {
+        title: 'Κίνδυνος',
+        deleteAccountButton: 'Διαγραφή λογαριασμού',
+        deleteAccountTitle: 'Διαγραφή λογαριασμού',
+        deleteAccountDescription: 'Διαγράψτε τον λογαριασμό σας και όλα τα συναφή δεδομένα',
+      },
+    },
+  },
+  profilePage: {
+    title: 'Ενημέρωση προφίλ',
+    imageFormTitle: 'Εικόνα προφίλ',
+    imageFormSubtitle: 'Ανέβασμα εικόνας',
+    imageFormDestructiveActionSubtitle: 'Αφαίρεση εικόνας',
+    fileDropAreaTitle: 'Σύρετε το αρχείο εδώ ή...',
+    fileDropAreaAction: 'Επιλέξτε αρχείο',
+    fileDropAreaHint: 'Ανεβάστε μια εικόνα σε μορφή JPG, PNG, GIF ή WEBP μικρότερη από 10 MB',
+    readonly:
+      'Οι πληροφορίες του προφίλ σας έχουν παρασχεθεί από τη σύνδεση με την επιχείρηση και δεν μπορούν να επεξεργαστούν.',
+    successMessage: 'Το προφίλ σας έχει ενημερωθεί.',
+  },
+  usernamePage: {
+    title: 'Ενημέρωση ονόματος χρήστη',
+    successMessage: 'Το όνομα χρήστη σας έχει ενημερωθεί.',
+  },
+  emailAddressPage: {
+    title: 'Προσθήκη διεύθυνσης email',
+    emailCode: {
+      formHint: 'Θα σταλεί ένα email που περιέχει έναν κωδικό επαλήθευσης σε αυτήν τη διεύθυνση email.',
+      formTitle: 'Κωδικός επαλήθευσης',
+      formSubtitle: 'Εισαγάγετε τον κωδικό επαλήθευσης που εστάλη στην {{identifier}}',
+      resendButton: 'Δεν λάβατε έναν κωδικό; Επανάληψη αποστολής',
```

Comment:
@EmmanouelaPothitou I think we can omit the "εναν" here. 
Result: "Δεν λάβατε κωδικό; Επανάληψη αποστολής"